### PR TITLE
Bump the doorbell timeout to 30 seconds

### PR DESCRIPTION
**Pull request by patrickod**

### CODE REVIEW

**Comment by hzeller** [comment]
30 seconds is ok for downstairs, but it might be an irritating user-feedback for the upstairs doors (upstairs and elevator), because people can hear the bell and might wonder why a second doorbell ring does not do anything (and we end up with destroyed buttons because people press it hard).

We need to think about a better user-interaction to make that more understandable but only once we have improved terminals (I want LCD displays on all of them, and multiple LEDs for rainbow or other effects).

Anyway, TL;DR: maybe we should have different auto-suppress times for gate (30s) and the rest (10s?).

**Comment by hzeller** [comment]
Are you still working on this ?
We might have different rate-limitings per door.
